packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
+    "build:ts": "rm -f migrations/*.js* && yarn ts-node --preferTsExts ./scripts/build.ts --truffleTypes ./types/typechain && tsc -b && mv migrations_ts/*.js* migrations",
+    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "check-versions": "./scripts/bash/check-versions.sh",
+    "ci:test-make-release": "./scripts/bash/release-on-devchain.sh",
     "clean": "rm -rf ./types/typechain && rm -rf build/* && rm -rf .0x-artifacts/* && rm -rf migrations/*.js* && rm -rf migrations_ts/*.js* && rm -rf test/**/*.js* && rm -f lib/*.js* && rm -f lib/**/*.js* && rm -f scripts/*.js* && yarn clean:foundry",
     "clean:foundry": "forge clean && rm -rf cache out",
-    "test": "rm test/**/*.js ; node runTests.js",
-    "test:scripts": "yarn ts-node scripts/run-scripts-tests.ts --testPathPattern=scripts/",
-    "quicktest": "./scripts/bash/quicktest.sh",
-    "test:coverage": "yarn run test --coverage",
-    "ci:test-make-release": "./scripts/bash/release-on-devchain.sh",
-    "test:release-snapshots": "./scripts/bash/release-snapshots.sh",
-    "test:generate-old-devchain-and-build": "./scripts/bash/generate-old-devchain-and-build.sh",
-    "build:ts": "rm -f migrations/*.js* && yarn ts-node --preferTsExts ./scripts/build.ts --truffleTypes ./types/typechain && tsc -b && mv migrations_ts/*.js* migrations",
-    "gas": "yarn run test --gas",
-    "pull-submodules": "git submodule update --init --recursive",
-    "delete-submodules": "rm -rf $(git submodule | awk '{ print $2 }')",
-    "build:sol": "yarn pull-submodules && mkdir -p migrations && yarn ts-node --preferTsExts ./scripts/build.ts --solidity ${BUILD_DIR:-./build}",
-    "build": "yarn build:sol && yarn build:ts",
-    "prebuild": "rm -rf ./build",
+    "console": "./scripts/bash/console.sh",
     "determine-release-version": "yarn --silent ts-node --preferTsExts ./scripts/determine-release-version.ts",
-    "prepare_contracts_and_abis_publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
-    "prepare_devchain_anvil_publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
-    "validate_abis_exports": "yarn ts-node --preferTsExts ./scripts/validate-abis-package.ts",
-    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
-    "migrate": "./scripts/bash/migrate.sh",
-    "set_block_gas_limit": "./scripts/bash/set_block_gas_limit.sh",
-    "download-artifacts": "./scripts/bash/download_artifacts.sh",
-    "init-network": "./scripts/bash/init_network.sh",
-    "upload-artifacts": "./scripts/bash/upload_artifacts.sh",
-    "upgrade": "./scripts/bash/upgrade.sh",
-    "revoke": "./scripts/bash/revoke.sh",
+    "devchain": "yarn ts-node scripts/devchain.ts",
+    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
+    "ganache-dev": "./scripts/bash/ganache.sh",
+    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
```

Comment:
Probably to remove with finishing migration from Hardhat to Foundry completely 😉